golang/testsuite/testsuite_impl/basic_datastore_and_api_test/basic_datastore_and_api_test_.go

```diff
@@ -59,15 +57,13 @@ func (b BasicDatastoreAndApiTest) Setup(networkCtx *networks.NetworkContext) (ne
 
 	logrus.Infof("Added datastore service with host port bindings: %+v", datastoreSvcHostPortBindings)
 
-	apiConfigFactory := api.NewApiContainerConfigFactory(b.apiImage, datastoreSvc)
-	uncastedApiService, apiSvcHostPortBindings, _, err := networkCtx.AddService(apiServiceId, apiConfigFactory)
+	apiConfigFactory := api.NewApiContainerConfigFactory(b.apiImage, datastoreServiceContext.GetIPAddress(), datastore.Port)
+	apiServiceInfo, apiSvcHostPortBindings, err := networkCtx.AddService(apiServiceId, apiConfigFactory)
```

Comment:
should `apiServiceInfo` be `apiServiceContext`?

Reply:
apiServiceContext
Good catch!! thanks